packages/commonwealth/server/routes/topics/create_topic_handler.ts

```diff
@@ -1,6 +1,12 @@
+import { AppError } from '../../../../common-common/src/errors';
 import { TopicAttributes } from '../../models/topic';
 import { ServerControllers } from '../../routing/router';
 import { TypedRequestBody, TypedResponse, success } from '../../types';
+import z from 'zod';
+
+const Errors = {
```

Comment:
Zod parsing will come up with appropriate error messages (see here for example: https://github.com/colinhacks/zod/blob/master/ERROR_HANDLING.md#a-demonstrative-example)

We could even define custom messages for individual rules (fields in the json) if the default ones aren't appropriate. I think throwing them individually this way will be more informative than aggregating them all into this ValidationError.